cmake/scripts/common/ModuleHelpers.cmake

```diff
@@ -0,0 +1,34 @@
+# This script provides helper functions for FindModules
+
+# Get archive name from VERSION dependency file
+# Arguments:
+#   name name of the library
+# On return:
+#   ARCHIVE will be set to parent scope
+#   MODULENAME_VER will be set to parent scope (eg FFMPEG_VER, DAV1D_VER)
+#   BASE_URL will be set to parent scope if in VERSION file
+function(get_archive_name module_name)
+  string(TOUPPER ${module_name} UPPER_MODULE_NAME)
+
+  # Dependency path
+  set(${UPPER_MODULE_NAME}_FILE "${CMAKE_SOURCE_DIR}/tools/depends/target/${module_name}/${UPPER_MODULE_NAME}-VERSION")
```

Comment:
Maybe bailout or log a warning if this file doesn't exist.

Reply:
Is there ever a case this could happen though? If src dir is stuffed up that much that its missing, its likely going to fail elsewhere?

My intent is to essentially have every dependency move to a VERSION file as standard, but as a stepping stone, only deps that can potentially be built from INTERNAL flags will use it for now

Reply:
Took your advice, added an exists check that will fatal error if the module VERSION isnt found. Better safe than sorry